### PR TITLE
Class20

### PR DESCRIPTION
**Pull request by yosef406**
this is a bit of a mess,
I moved the main branch from the repository yosef-awad.
now i have added some documentation for the JavaScript.

### CODE REVIEW

**Comment by yosef406** [comment]
removed the conflicts

File: script/script.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
+/**
+ * @param {number} sum used to save the sum of all the grades
```
Comment (EliasSalom) [inline script/script.js:2]: what is the point behind `@param`?

**Review by EliasSalom — CHANGES_REQUESTED** [review]
I think you have copied this code from the internet!!!

File: script/script.js (hunk 1 above)
Comment (yosef406) [reply script/script.js:2]: the @param is to identify that 'sum' is a parameter of type {number}.
it is created automatically by vsCode by typing /** 
the correction of the syntax was from this website [link](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html)
specifically the part @param [link](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns)

**Review by EliasSalom — APPROVED** [review]
Good work

**Pull request merged**